app/src/main/java/ch/epfl/favo/view/tabs/MapPage.java

```diff
@@ -423,9 +485,11 @@ public void onInfoWindowClick(Marker marker) {
       Navigation.findNavController(view)
           .navigate(R.id.action_nav_map_to_favorEditingView, favorBundle);
     } else {
-      favorBundle.putString(CommonTools.FAVOR_ARGS, favorId);
-      Navigation.findNavController(view)
-          .navigate(R.id.action_nav_map_to_favorPublishedView, favorBundle);
+      if (isRequested) {
```

Comment:
this condition check should not appear, otherwise we cannot click favors posted by others

Reply:
You're right, I initially modified it in order to prevent that when choosing the location the user could click on the info window.